GUI/Types/Exporter/ExportFile.cs

```diff
@@ -119,6 +120,16 @@ public static void ExtractFileFromStream(string fileName, Stream stream, VrfGuiC
             }
             else
             {
+                if (decompile && fileName.EndsWith(".vfe", StringComparison.OrdinalIgnoreCase))
```

Comment:
Maybe move this to `FileExtract` so that cli will also be able to use this with minimal changes (and then we can pull specific code from cli into this class as well so gui can export non resource files)